### PR TITLE
update gemspec and spec to use Rails 3.2 instead of 2.3

### PR DESCRIPTION
**Pull request by jcarres-mdsol**
I think this solves the problem with the specs.
New rubies do not work with Rails 2.3.x so I've updated to 3.2 which I guess is the minimum we support.

Also cleaned up gemspec and Gemfile while at it
@ejennings-mdsol

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -23,6 +19,8 @@ group :development do
   end
 end
 
-gem 'activesupport', '~> 2.3', require: 'active_support'
+#gem 'activesupport', '~> 2.3', require: 'active_support'
```
Comment (ejennings-mdsol) [inline Gemfile]: Can this line be deleted?

**Comment by ejennings-mdsol** [comment]
Travis is failing.  Not sure if adding
```
before_install:
  - gem update bundler
```
to  .travis.yml would help.

**Comment by ejennings-mdsol** [comment]
Thanks @jcarres-mdsol!  Travis is happy now!

**Comment by sakhtermdsol** [comment]
Hey @jcarres-mdsol thanks for this. I was wondering since this isn't backwards compatible with previous Rails/Ruby versions, wouldn't this warrant a Major version increase? PR looks good to me otherwise.

**Comment by jcarres-mdsol** [comment]
@sakhtermdsol  In fact the changes are all in the testing infrastructure, there is no code changes that could affect the users so it should be backwards compatible.
The test gems did not work with modern rubies and we do not test old ones. As there are no functional changes have not even updated the minor version.

**Comment by sakhtermdsol** [comment]
Ah I see. No worries. Thanks!

**Pull request merged**